lib/tapioca/dsl/compilers/aasm.rb

```diff
@@ -1,12 +1,7 @@
 # typed: strict
 # frozen_string_literal: true
 
-begin
-  require "active_record"
-  require "aasm"
-rescue LoadError
-  return
-end
+return unless defined?(AASM)
```

Comment:
This was requiring Active Record as well. Does the compiler need `ActiveRecord` to be defined too?